src/Microsoft.TemplateEngine.Edge/Microsoft.TemplateEngine.Edge.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj" />
     <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj" />
+    <PackageReference Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1Version)" />
```

Comment:
```suggestion
    <PackageReference Include="System.Formats.Asn1" Version="$(SystemFormatsAsn1Version)" Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Reply:
We did that for Microsoft.Redist.IO but I don't think that should be required for this since I'm using the net8 version of this.